PanCardView/Enums/StripePostion.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace PanCardView.Enums
+{
+    public enum StripePosition
+    {
+        Top,
```

Comment:
Bottom,
Top

Please change the order of elements (Default value is Bottom, so it should be the first element)